README.md

```diff
@@ -25,53 +25,157 @@
+
+Add the component with some nodes and edges:
 
 ```jsx
 import React from 'react';
 import { Canvas } from 'reaflow';
 
 export default () => (
-  <Canvas
-    nodes={[
-      {
-        id: '1',
-        text: '1'
-      },
-      {
-        id: '2',
-        text: '2'
-      }
-    ]}
-    edges={[
-      {
-        id: '1-2',
-        from: '1',
-        to: '2'
-      }
-    ]}
-  />
+  <div style={{ position: 'relative', width: '100vw', height: '100vh' }}>
+    <div style={{ position: 'absolute', top: 0, bottom: 0, left: 0, right: 0, 'background-color': 'lightgrey' }}>
+      <Canvas
+        maxWidth={800} // Use small canvas dimensions to make sure the nodes appear on screen immediately
+        maxHeight={600}
+        nodes={[
+          {
+            id: '1',
+            text: '1'
+          },
+          {
+            id: '2',
+            text: '2'
+          }
+        ]}
+        edges={[
+          {
+            id: '1-2',
+            from: '1',
+            to: '2'
+          }
+        ]}
+      />
+    </div>
+  </div>
 );
 ```
 
-## 🔭 Developing
+## Usage with Next.js framework
+
+The Next.js framework supports SSR, but the `Canvas` component shouldn't be rendered on the server.
+_It won't crash the app, but it'd print a lot of noisy warnings on the server, though._
+
+`pages/index.tsx`:
+```tsx
+import React from 'react';
+import { Canvas } from 'reaflow';
+
+const Page = () => {
+  return (
+    <div style={{position: 'relative', width: '100vw', height: '100vh'}}>
+      <div style={{position: 'absolute', top: 0, bottom: 0, left: 0, right: 0, 'backgroundColor': '#F5F5F5'}}>
+        {
+          // Don't render the Canvas on the server
+          typeof window !== 'undefined' && (
+            <Canvas
+              maxWidth={800} // Use small canvas dimensions to make sure the nodes appear on screen immediately
+              maxHeight={600}
+              nodes={[
+                {
+                  id: '1',
+                  text: '1'
+                },
+                {
+                  id: '2',
+                  text: '2'
+                }
+              ]}
+              edges={[
+                {
+                  id: '1-2',
+                  from: '1',
+                  to: '2'
+                }
+              ]}
+            />
+          )
+        }
+      </div>
+    </div>
+  )
+};
+
+export default Page;
+```
+
+## Community showcases
+
+- [POC Next.js + Reaflow](https://github.com/Vadorequest/poc-nextjs-reaflow): Uses Reaflow to build a "**decision tree**". 
+  Advanced use-case with different kinds of nodes with heavy usage of `foreignObject` to display actual HTML within the nodes themselves.
+  Hosted on Vercel and built with Next.js 10.
+
+---
+
+# 🔭 Contributing
+
+## Getting started
 
 - `yarn install`
-- `yarn start`
-- Storybook Opens
+- `yarn start` opens Storybook documentation on [localhost:9009](http://localhost:9009)
+
+## Using the `reaflow` package locally as a dependency
```

Comment:
Let's move this under Getting Started / Install page

Reply:
Maybe better "Contributing"?

Reply:
Ya - that sounds good